sideci_demo/phpmd_demo.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Github\Client;
+
+class Repository
```

Comment:
__[PHP_CodeSniffer]__

Each class must be in a namespace of at least one level (a top-level vendor name)


<!-- Comment by [SideCI](https://sideci.com) -->